balloon.go

```diff
@@ -0,0 +1,78 @@
+package firecracker
```

Comment:
Please add `Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.` header as like other files. We should have an automated check for that though.

Reply:
Fixed it, but this [drivers.go](https://github.com/firecracker-microvm/firecracker-go-sdk/blob/master/drives.go) also does not contain he copyright statement, should I add for it?

Reply:
Ah okay. We can fix others in a different pull request later.